sdk/core/src/CryptoLib/README.md

```diff
@@ -0,0 +1,3 @@
+# Description
+
+This is a rebuilding of the library [eccrypto-js](https://github.com/pedrouid/eccrypto-js) mainly to polyfill encryption/decryption method for react-native environment.
```

Comment:
if it is for react-native / browser, why it is in core-sdk? Maybe it makes more sense to move to concrete SDK where it is going to be consumed?

Reply:
Moved changes into sdk/react-native. Changes were only required for react-native env as original third-party lib was build for browser/node env.